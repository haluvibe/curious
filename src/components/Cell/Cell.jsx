import React from "react";
import Grid from "material-ui/Grid";

function Cell (props) {
  const { children, ...other } = props;

  return (
    <Grid item {...other}>
      {children}
    </Grid>
  );
}

Cell.defaultProps = {
  xs: 12
};

export default Cell;
