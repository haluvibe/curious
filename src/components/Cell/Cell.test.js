import React from "react";
import toJson from "enzyme-to-json";
import { shallow } from "enzyme";

import Cell from "./Cell";

test("Cell component should render as expected", () => {
  const component = shallow(<Cell />),
    tree = toJson(component);
  expect(tree).toMatchSnapshot();
});
