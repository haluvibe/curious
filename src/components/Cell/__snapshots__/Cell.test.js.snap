// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell component should render as expected 1`] = `
<WithStyles(GridWrapper)
  item={true}
  xs={12}
/>
`;
